modules/tinymce/src/themes/silver/main/ts/api/Events.ts

```diff
@@ -5,8 +5,8 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { UIEvent } from '@ephox/dom-globals';
 import Editor from 'tinymce/core/api/Editor';
+import { Event } from '@ephox/dom-globals';
```

Comment:
This should be above `tinymce/...` as it's an external import (probably doesn't matter for master, but it will fail on develop).